tests/app/main/views/test_suppliers.py

```diff
@@ -295,122 +280,34 @@ def test_details_link_is_shown_to_users_with_right_roles(self, role, link_should
             "Role {} {} see the link".format(role, "can not" if link_should_be_visible else "can")
         )
 
-    @pytest.mark.parametrize("role, links_should_be_visible", [
-        ("admin", False),
-        ("admin-ccs-category", False),
-        ("admin-ccs-sourcing", True),
-        ("admin-framework-manager", False),
-    ])
-    def test_declaration_and_agreement_links_visible_for_ccs_sourcing(self, role, links_should_be_visible):
-        self.user_role = role
-        self.data_api_client.find_frameworks.return_value = {
-            'frameworks': [
-                FrameworkStub(slug='g-cloud-7', name='G-Cloud 7', id=1).response(),
-                FrameworkStub(slug='g-cloud-10', id=2).response(),
-            ]
-        }
-        response = self.client.get('/admin/suppliers?supplier_name=foo')
-        data = response.get_data(as_text=True)
-        document = html.fromstring(data)
-
-        framework_column_entries = [framework.strip() for framework in document.xpath('//tr/td/span/text()')]
-        assert links_should_be_visible is (framework_column_entries == ['G-Cloud 10', 'G-Cloud 7'])
-
-        if links_should_be_visible:
-            g_cloud_10_edit_declaration = ''.join(document.xpath('//tbody//tr[1]/td[2]//a//text()')).strip()
-            assert g_cloud_10_edit_declaration == 'Edit declaration for G-Cloud 10'
-
-            g_cloud_10_view_agreement = ''.join(document.xpath('//tbody//tr[1]/td[3]//a//text()')).strip()
-            assert g_cloud_10_view_agreement == 'View agreement for G-Cloud 10'
-
-            g_cloud_7_edit_declaration = ''.join(document.xpath('//tbody//tr[2]/td[2]//a//text()')).strip()
-            assert g_cloud_7_edit_declaration == 'Edit declaration for G-Cloud 7'
-
-            g_cloud_7_download_agreement = ''.join(document.xpath('//tbody//tr[2]/td[3]//a//text()')).strip()
-            assert g_cloud_7_download_agreement == 'Download agreement for G-Cloud 7'
-
-            g_cloud_7_download_signed_agreement = ''.join(document.xpath('//tbody//tr[2]/td[4]//a//text()')).strip()
-            assert g_cloud_7_download_signed_agreement == 'Download signed agreement for G-Cloud 7'
-
-            g_cloud_7_upload_countrsigned_agreement = ''.join(document.xpath('//tbody//tr[2]/td[5]//a//text()')).strip()
-            assert g_cloud_7_upload_countrsigned_agreement == 'Upload countersigned agreement for G-Cloud 7'
-        else:
-            for text in [
-                'Edit declaration',
-                'View agreement',
-                'Download agreement',
-                'Download signed agreement',
-                'Upload countersigned agreement'
-            ]:
-                assert not document.xpath("//tbody//text()='$text'", text=text)
-
-    def test_agreements_template_only_lists_frameworks_that_we_care_about_in_reverse_order(self):
```

Comment:
Ah. This test reminds me that CCS Sourcing users see a different subset of frameworks than the other admin roles.

CCS Sourcing are interested in all framework statuses except `coming` (see https://github.com/alphagov/digitalmarketplace-admin-frontend/blob/master/app/main/views/suppliers.py#L74 for where the 'interesting_frameworks' template var is populated). 

In the details page, we're only showing `live` and `expired` (see https://github.com/alphagov/digitalmarketplace-admin-frontend/blob/master/app/main/views/suppliers.py#L132), as that's what CCS Category and the other admin roles care about. So we'll have to tweak that view to ensure sourcing can see links for `open`, `pending` and `standstill` frameworks.

Reply:
I think I fixed this with 5c6c5b1